diff --git a/node_modules/evergreen-ui/esm/dialog/src/Dialog.js b/node_modules/evergreen-ui/esm/dialog/src/Dialog.js
index 968ba67..cea29d8 100644
--- a/node_modules/evergreen-ui/esm/dialog/src/Dialog.js
+++ b/node_modules/evergreen-ui/esm/dialog/src/Dialog.js
@@ -136,8 +136,7 @@ function (_React$Component) {
             close = _ref.close;
         return React.createElement(Pane, _extends({
           role: "dialog",
-          backgroundColor: "white",
-          elevation: 4,
+          backgroundColor: "#202020",
           borderRadius: 8,
           width: width,
           maxWidth: maxWidth,
@@ -171,7 +170,10 @@ function (_React$Component) {
           padding: 16,
           flexDirection: "column",
           minHeight: minHeightContent
-        }, contentContainerProps), React.createElement(Pane, null, _this2.renderChildren(close))), hasFooter && React.createElement(Pane, {
+        }, contentContainerProps), React.createElement(Pane, {
+          overflow: "hidden",
+          display: "flex"
+        }, _this2.renderChildren(close))), hasFooter && React.createElement(Pane, {
           borderTop: "muted",
           clearfix: true
         }, React.createElement(Pane, {
@@ -348,7 +350,6 @@ _defineProperty(Dialog, "defaultProps", {
   hasFooter: true,
   hasCancel: true,
   intent: 'none',
-  width: 560,
   topOffset: '12vmin',
   sideOffset: '16px',
   minHeightContent: 80,
diff --git a/node_modules/evergreen-ui/esm/theme/src/default-theme/component-specific/overlayBackgroundColor.js b/node_modules/evergreen-ui/esm/theme/src/default-theme/component-specific/overlayBackgroundColor.js
index 6faad6f..a863514 100644
--- a/node_modules/evergreen-ui/esm/theme/src/default-theme/component-specific/overlayBackgroundColor.js
+++ b/node_modules/evergreen-ui/esm/theme/src/default-theme/component-specific/overlayBackgroundColor.js
@@ -1,4 +1,4 @@
 import scales from '../foundational-styles/scales';
-var overlayBackgroundColor = scales.neutral.N7A;
+var overlayBackgroundColor = "rgba(0,0,0,0.8)";
 export default overlayBackgroundColor;
 //# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbIi4uLy4uLy4uLy4uLy4uL3NyYy90aGVtZS9zcmMvZGVmYXVsdC10aGVtZS9jb21wb25lbnQtc3BlY2lmaWMvb3ZlcmxheUJhY2tncm91bmRDb2xvci5qcyJdLCJuYW1lcyI6WyJzY2FsZXMiLCJvdmVybGF5QmFja2dyb3VuZENvbG9yIiwibmV1dHJhbCIsIk43QSJdLCJtYXBwaW5ncyI6IkFBQUEsT0FBT0EsTUFBUCxNQUFtQiwrQkFBbkI7QUFFQSxJQUFNQyxzQkFBc0IsR0FBR0QsTUFBTSxDQUFDRSxPQUFQLENBQWVDLEdBQTlDO0FBRUEsZUFBZUYsc0JBQWYiLCJzb3VyY2VzQ29udGVudCI6WyJpbXBvcnQgc2NhbGVzIGZyb20gJy4uL2ZvdW5kYXRpb25hbC1zdHlsZXMvc2NhbGVzJ1xuXG5jb25zdCBvdmVybGF5QmFja2dyb3VuZENvbG9yID0gc2NhbGVzLm5ldXRyYWwuTjdBXG5cbmV4cG9ydCBkZWZhdWx0IG92ZXJsYXlCYWNrZ3JvdW5kQ29sb3JcbiJdfQ==
\ No newline at end of file
diff --git a/node_modules/evergreen-ui/esm/theme/src/default-theme/typography/fontFamilies.js b/node_modules/evergreen-ui/esm/theme/src/default-theme/typography/fontFamilies.js
index 3bd061d..0b2ed3a 100644
--- a/node_modules/evergreen-ui/esm/theme/src/default-theme/typography/fontFamilies.js
+++ b/node_modules/evergreen-ui/esm/theme/src/default-theme/typography/fontFamilies.js
@@ -2,12 +2,12 @@ export default {
   /**
    * @property {string} display - Used for headings larger than 20px.
    */
-  display: "\"SF UI Display\", -apple-system, BlinkMacSystemFont, \"Segoe UI\", Roboto, Helvetica, Arial, sans-serif, \"Apple Color Emoji\", \"Segoe UI Emoji\", \"Segoe UI Symbol\"",
+  display: "\"Voces\", -apple-system, BlinkMacSystemFont, \"Segoe UI\", Roboto, Helvetica, Arial, sans-serif, \"Apple Color Emoji\", \"Segoe UI Emoji\", \"Segoe UI Symbol\"",
 
   /**
    * @property {string} ui - Used for text and UI (which includes almost anything).
    */
-  ui: "\"SF UI Text\", -apple-system, BlinkMacSystemFont, \"Segoe UI\", Roboto, Helvetica, Arial, sans-serif, \"Apple Color Emoji\", \"Segoe UI Emoji\", \"Segoe UI Symbol\"",
+  ui: "\"Voces\", -apple-system, BlinkMacSystemFont, \"Segoe UI\", Roboto, Helvetica, Arial, sans-serif, \"Apple Color Emoji\", \"Segoe UI Emoji\", \"Segoe UI Symbol\"",
 
   /**
    * @property {string} mono - Used for code and sometimes numbers in tables.
